audnauseum/gui/ui.py

```diff
@@ -120,6 +128,18 @@ def open_file_dialog(ui) -> str:
     return file_path
 
 
+# Modified from source code example: https://pythonspot.com/pyqt5-file-dialog/
+
+def save_file_dialog(ui) -> str:
+    options = QFileDialog.Options()
+    options |= QFileDialog.DontUseNativeDialog
+
+    file_path, _ = QFileDialog.getSaveFileName(
+        ui, "QFileDialog.getSaveFileName()", "", "Loops Files (*.json)", options=options)
```

Comment:
Suggestion - update the title of the dialog to something more user-friendly and open the dialog to the "resources/json" folder.
```suggestion
        ui, "Save a Loop File", "./resources/json", "Loops Files (*.json)", options=options)
```

Reply:
Agreed, I will fix that in the load and save dialogs